projects/plugins/social/declaration.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module '*.module.css' {
+	const classes: { [ key: string ]: string };
+	export default classes;
+}
+
+declare module '*.module.scss' {
+	const classes: { [ key: string ]: string };
+	export default classes;
+}
```

Comment:
I had some errors, because with TypeScript you can only import valid modules, and without this the usual scss import we do `import styles from './styles.module.scss';` would show an error, because it cannot recognise syntax there. This defines that the scss files are generally arrays that have named objects with string key-value pairs. I found this type of solution in other parts of the repo too (like in the js-packages folder).

Reply:
Update: I ended up using the bas tsconfig instead which already does this